pystac/extensions/version.py

```diff
@@ -22,14 +22,6 @@
 VERSION: str = "version"
 DEPRECATED: str = "deprecated"
 
-# Link "rel" attribute values.
-LATEST: str = "latest-version"
-PREDECESSOR: str = "predecessor-version"
-SUCCESSOR: str = "successor-version"
```

Comment:
Maybe it makes sense to add extension-specific relation types to a comment or docstring somewhere, to make it easier for users looking at the version source for the first time to see what RelTypes this extension adds? The constants-on-top approach the extensions have been nice for serving as a documentation of the spec - you see it adds these reltypes and you end up groking a bit more about the extension spec. Moving the extension rel types to RelType makes sense to me, as it keeps it all accessible in one place, and some source file docs would balance out the only negative effect I see of that change.